tests/integration/cattletest/core/test_allocation.py

```diff
@@ -229,7 +232,8 @@ def test_host_vnet_association(admin_client, sim_context,
 
 
 def test_allocation_stay_associated_to_host(admin_client, sim_context):
-    c = admin_client.create_container(imageUuid=sim_context['imageUuid'])
+    c = admin_client.create_container(
```

Comment:
Just fixed it. It happened because I changed it back to use "admin_client" from "legacyAdminClient" which made the number of characters in this line to less than 80.